site/oauth2-examples-oauth0.md

```diff
@@ -0,0 +1,134 @@
+# Use https://auth0.com/ as OAuth 2.0 server
+
+We are going to test 3 OAuth flows:
+1. Access management ui via a browser
+2. Access management rest api
+3. Access AMQP protocol
+
+## Prerequisites to follow this guide
+
+* Have an account in https://auth0.com/.
+* Docker
+
+## Create rabbitmq API
+
+In OAuth0, resources are mapped to Application APIs. Once you have logged onto your account in https://auth0.com/, go to dashboard > Applications > APIs > Create an API.
+
```

Comment:
Should the UI sequence "dashboard > Applications > APIs > Create an API" be bolded to be consistent with all other UI paths referenced in this doc